workflows/chipseq/chipseq-standard.wdl

```diff
@@ -152,7 +151,7 @@ task parse_input {
 
     runtime {
         disk: "1 GB"
-        docker: 'ghcr.io/stjudecloud/util:1.1.0'
+        docker: 'ghcr.io/stjudecloud/util:1.0.0'
```

Comment:
Similar question here, why are we rolling back?

Reply:
In my opinion, we should use the latest here regardless of the size. I think this will be too difficult to keep track of why we did what otherwise.

Reply:
I don't have a problem with bumping it to the latest. I'll make the change.